tests/loggers/test_all.py

```diff
@@ -102,7 +102,7 @@ def test_epoch_end(self, outputs) -> None:
 
     class StoreHistoryLogger(logger_class):
 
-        def __init__(self, *args, **kwargs):
+        def __init__(self, *args, **kwargs) -> None:
```

Comment:
I do not think that init are annotated with None cc: @carmocca 
```suggestion
        def __init__(self, *args, **kwargs):
```

Reply:
removed !

Reply:
everywhere? bc it was on about 10 other places in this PR